packages/platform-sdk-ark/src/networks/ark/devnet.ts

```diff
@@ -1,37 +1,69 @@
 import { Coins } from "@arkecosystem/platform-sdk";
 
 const network: Coins.NetworkManifest = {
-	id: "ark.devnet",
+	uuid: "ark.devnet",
```

Comment:
What was wrong with `id`? And this... is definitely not a `uuid`.

Reply:
A few networks have a literal parameter called network ID so would want to avoid confusion which network ID is meant.

Reply:
Still think uuid is not a good idea. It's misleading.

Reply:
Suggest a better name :trollface:

Reply:
I'm also for avoid uuid.

> confusion which network ID

Call it network name?

Reply:
It's not a network name, that's `Devnet` in this case. It's a universally unique identifier **(there's no 2 ark.devnet across any implementation)** for the network based on it's coin and name.